From 361cefcc6c5df1a49adcb3135634ddd6fdcc3d8f Mon Sep 17 00:00:00 2001
From: Haoran Wang <b50027@freescale.com>
Date: Wed, 8 Jul 2015 10:23:00 +0800
Subject: [PATCH] MA-6854 Remove BT_SDIO support

Remove BT_SDIO for BCM4339.
As the BT_SDIO will caused bcm4339 panic when suspend.

Signed-off-by: Haoran Wang <b50027@freescale.com>
---
 arch/arm/configs/imx_v7_android_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/configs/imx_v7_android_defconfig b/arch/arm/configs/imx_v7_android_defconfig
index 4e4ad8b..66b3e47 100644
--- a/arch/arm/configs/imx_v7_android_defconfig
+++ b/arch/arm/configs/imx_v7_android_defconfig
@@ -181,7 +181,6 @@ CONFIG_BT_BNEP_MC_FILTER=y
 CONFIG_BT_BNEP_PROTO_FILTER=y
 CONFIG_BT_HIDP=y
 CONFIG_BT_HCIBTUSB=y
-CONFIG_BT_HCIBTSDIO=y
 CONFIG_BT_HCIUART=y
 CONFIG_BT_HCIUART_BCSP=y
 CONFIG_BT_HCIUART_ATH3K=y
-- 
1.8.0

